l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -69,7 +69,7 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
     mapping(uint256 chainId => uint256 activeSettlementLayerChainId) public settlementLayer;
 
     /// @dev Sync layer chain is expected to have .. as the base token.
-    mapping(uint256 chainId => bool isWhitelistedSyncLayer) public whitelistedSettlementLayers;
+    mapping(uint256 chainId => bool isWhitelistedSettlementLayer) public whitelistedSettlementLayers;
```

Comment:
please add a comment that `Gateway` is just one of the possible settlement layers. The main point of confusion was interchangeable use of "sync layer" and "settlement layer". I think this PR does not solve the issue completely as to someone the difference may not be clear